templates/actors/limited-sheet.html

```diff
@@ -14,8 +14,7 @@ <h1 class="charname">
     {{!-- Sheet Body --}}
     <section class="sheet-body">
         <div class="tab biography">
-            <p>Biography</p>
-            {{editor content=data.details.biography.value target="data.details.biography.value" button=true owner=owner editable=editable}}
+            {{editor content=data.biography target="data.biography" button=true owner=owner editable=editable}}
```

Comment:
There is duplication of the biography. I would extract that to a part and reuse it.

Reply:
As per the description

Reply:
I went through it again, since the biography is only taking one line to set, I prefer avoiding splitting everything in subtemplates as I don't know the cost to invoke a subtemplate yet. I prefer use subtemplates when things get a bit complicated, for instance maybe the Peritia ? I don't know to be honest